app/models/container_image.rb

```diff
@@ -6,4 +6,12 @@ class ContainerImage < ActiveRecord::Base
   has_many :containers
 
   acts_as_miq_taggable
+
+  def full_name
+    result = ""
+    result << "#{container_image_registry.full_name}/" unless container_image_registry.nil?
+    result << name
+    result << ":#{tag}" unless tag.nil?
```

Comment:
I don't know of a better way of doing this right now due to the conditional values + "/" and ":" but the series of shovel operations feels complicated.  Maybe we can look at it in a follow up PR.